### PR TITLE
Complement Apache APISIX architecture documentation with Basic Object Relationships

### PR DESCRIPTION
**Pull request by Miss-you**
Complement Apache APISIX architecture documentation with its basic object relationships
Make it easier for everyone to understand Route and other objects

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: doc/architecture-design-cn.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 ## 目录
 - [**APISIX**](#apisix)
 - [**APISIX Config**](#apisix-config)
+- [**APISIX对象纵览**](#APISIX对象纵览)
```
Comment (membphis) [inline doc/architecture-design-cn.md]: missing space between Chinese and English

File: doc/architecture-design-cn.md [hunk 2]
```diff
@@ -64,6 +65,20 @@ plugins:                        # plugin name list
 
 [返回目录](#目录)
 
+## APISIX对象纵览
+
+![](./images/object-relation.png)
+
+1. Route 是路由，通过定义一些规则来匹配客户端的请求，并加载相应插件以及转发到指定后端 Upstream 。Route 可以等价认为是一个 API 或者匹配规则。
```
Comment (membphis) [inline doc/architecture-design-cn.md]: One route can contain more than one URI.

File: doc/architecture-design-cn.md [hunk 3]
```diff
@@ -64,6 +65,20 @@ plugins:                        # plugin name list
...
+2. Service 是 API 的组，多个 Route 对应一个 Service 。与 Route 对应关系：n 对 1
```
Comment (moonming) [inline doc/architecture-design-cn.md]: I think `route`,`uptream`, `plugin` have been explained clearly in their own section.

**Comment by moonming** [comment]
this PR that hasn't been updated for a long time, close it first

**Pull request closed without merging**